src/libraries/System.Text.Json/src/System/Text/Json/Serialization/ReflectionEmitMemberAccessor.cs

```diff
@@ -27,6 +31,9 @@ internal sealed class ReflectionEmitMemberAccessor : MemberAccessor
                 return null;
             }
 
+#if NET6_0 // should really be NET6_0_OR_GREATER
+            return new JsonClassInfo.ConstructorDelegate(Activator.CreateFactory(type));
```

Comment:
There are 20 instance of ConstructorDelegate in 7 files. I think we should just replace it with `Func<object>`.